iris-mpc-cpu/src/hawkers/shared_irises.rs

```diff
@@ -10,7 +10,8 @@ use crate::execution::hawk_main::state_check::SetHash;
 /// Storage of inserted irises.
 #[derive(Clone, Debug, Serialize, Deserialize, PartialEq)]
 pub struct SharedIrises<I: Clone> {
-    pub points: HashMap<SerialId, (VersionId, I)>,
+    pub points: Vec<Option<(VersionId, I)>>,
```

Comment:
Just wondering if that could be not-pub now?

Reply:
I made it private and added a getter.